src/main/java/org/broadinstitute/hellbender/tools/copynumber/PostprocessGermlineCNVCalls.java

```diff
@@ -348,7 +350,8 @@ public Object onTraversalSuccess() {
 
     private void generateIntervalsVCFFileFromAllShards() {
         logger.info("Generating intervals VCF file...");
-        final VariantContextWriter intervalsVCFWriter = createVCFWriter(outputIntervalsVCFFile);
+        final VariantContextWriter intervalsVCFWriter = GATKVariantContextUtils.createVCFWriter(
```

Comment:
This is strange, because createVCFWriter creates output indexes by default.

Reply:
Oh... it's because the sequence dictionary is weird in this tool.  It might make sense to fix this tools implementation of getBestAvailableSequenceDictionary instead of changing these calls, but maybe that's harder.

Reply:
It's because of the way the tool takes inputs.  It takes a parent directory that contains the files, so there's no single file the engine knows how to get a sequence dictionary from.  I hate to change the input arguments at this point, though.

Reply:
We could potentially override getBestAvailableSequenceDictionary in this tool to understand what this tool uses, and then other stuff would *JustWork*